x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/use_alerts_flyout.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  AlertsTableFlyoutBaseProps,
+  AlertTableFlyoutComponent,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import { get } from 'lodash';
+import React from 'react';
+import { EuiDescriptionList, type EuiDataGridColumn, EuiPanel, EuiTitle } from '@elastic/eui';
+import { ALERT_RULE_NAME } from '@kbn/rule-data-utils';
+import { RegisterFormatter } from './render_cell_value';
+
+const FlyoutHeader: AlertTableFlyoutComponent = ({ alert }: AlertsTableFlyoutBaseProps) => {
+  const name = alert[ALERT_RULE_NAME];
+  return (
+    <EuiTitle size="s">
+      <h3>{name}</h3>
+    </EuiTitle>
+  );
+};
+
+export const getAlertFlyout =
```

Comment:
When I try and open the alert flyout for the Active alerts here:

<img width="662" alt="image" src="https://github.com/elastic/kibana/assets/7405507/c705035e-b622-4c07-a913-8310245d501e">

The page crashes with this error:

<img width="495" alt="image" src="https://github.com/elastic/kibana/assets/7405507/7c44d118-9373-4942-81e8-f65b290e9cd9">

Reply:
Tested fixed by https://github.com/elastic/kibana/pull/167998/commits/6a638ecb2cc4518df1978281e7f31139c1466da2